src/System Application/App/Privacy Notice/src/PrivacyNoticeImpl.Codeunit.al

```diff
@@ -23,20 +23,20 @@ codeunit 1565 "Privacy Notice Impl."
         MissingLinkErr: Label 'No privacy notice link was specified';
         PrivacyNoticeDoesNotExistErr: Label 'The privacy notice %1 does not exist.', Comment = '%1 = the identifier of a privacy notice';
         TelemetryCategoryTxt: Label 'Privacy Notice', Locked = true;
-        CreatePrivacyNoticeTelemetryTxt: Label 'Creating privacy notice', Locked = true;
-        ConfirmPrivacyNoticeTelemetryTxt: Label 'Confirming privacy notice', Locked = true;
-        PrivacyNoticeAutoApprovedByAdminTelemetryTxt: Label 'The privacy notice was auto-approved by the admin', Locked = true;
-        PrivacyNoticeAutoRejectedByAdminTelemetryTxt: Label 'The privacy notice was auto-rejected by the admin', Locked = true;
-        PrivacyNoticeAutoApprovedByUserTelemetryTxt: Label 'The privacy notice was auto-approved by the user', Locked = true;
-        ShowingPrivacyNoticeTelemetryTxt: Label 'Showing privacy notice', Locked = true;
-        PrivacyNoticeApprovalResultTelemetryTxt: Label 'Approval State after showing privacy notice: %1', Locked = true;
-        CheckPrivacyNoticeApprovalStateTelemetryTxt: Label 'Checking privacy approval state', Locked = true;
-        AdminPrivacyApprovalStateTelemetryTxt: Label 'Admin privacy approval state: %1', Locked = true;
-        UserPrivacyApprovalStateTelemetryTxt: Label 'User privacy approval state: %1', Locked = true;
+        CreatePrivacyNoticeTelemetryTxt: Label 'Creating privacy notice %1', Locked = true;
```

Comment:
The privacy notice table (`table 2000000237 "Privacy Notice"`) is a system table in our platform that partners can use as well.
Here we are effectively treating the ID of that table as System Metadata, but this might not be immediately clear to partners that might be using the table.

I suggest we also make a change to the system table `table 2000000237 "Privacy Notice"` (separate to this PR) to mark the ID field with the annotation `DataClassification = SystemMetadata;`, just to make the classification explicit.